permissions/generators_static_data.rb

```diff
@@ -153,51 +153,49 @@ def spaced_to_pascal(spaced)
  *
  * https://imqssoftware.atlassian.net/wiki/display/ASC/Generating+Permissions
  */
-namespace iq {
-	export namespace auth {
-
-		export class Permission {
-			id: string;
-			name: string;
-			friendlyName: string;
-			description: string;
-			module: AuthModule;
-
-			constructor(id: string, name: string, friendlyName: string, description: string, module: AuthModule) {
-				this.id = id;
-				this.name = name;
-				this.friendlyName = friendlyName;
-				this.description = description;
-				this.module = module;
-			}
-		}
+import { AuthModule } from './modules';
+import { LocalStorageKeys } from "../js-base/defs";
+
+export class Permission {
+	id: string;
+	name: string;
+	friendlyName: string;
+	description: string;
+	module: AuthModule;
+
+	constructor(id: string, name: string, friendlyName: string, description: string, module: AuthModule) {
+		this.id = id;
+		this.name = name;
+		this.friendlyName = friendlyName;
+		this.description = description;
+		this.module = module;
+	}
+}
 
-		// List of permissions in imqsauth project (search for 'PermissionsTable')
-		export const Permissions = {
+// List of permissions in imqsauth project (search for 'PermissionsTable')
+export const Permissions = {
 ENUMS
-		};
-
-		/** @suppress {suspiciousCode} */
-		export let permissionsArray: iq.auth.Permission[];
-
-		// populate RoleArray
-		(function () {
-			iq.auth.permissionsArray = [];
-			for (let key in iq.auth.Permissions) {
-				if (iq.auth.Permissions.hasOwnProperty(key)) {
-					iq.auth.permissionsArray.push(iq.auth.Permissions[key]);
-				}
-			}
-		}());
-
-		export function getPermissionByID(id: string): iq.auth.Permission {
-			for (let i = 0; i < iq.auth.permissionsArray.length; i++) {
-				if (iq.auth.permissionsArray[i].id === id)
-					return iq.auth.permissionsArray[i];
-			}
-			return null;
+};
+
+/** @suppress {suspiciousCode} */
+export let permissionsArray: Permission[];
+
+// populate RoleArray
+(function () {
+	permissionsArray = [];
+	for (let key in Permissions) {
```

Comment:
Change the `let` to a const.
Drop the braces for the `for loop` and the `if` statement.